### PR TITLE
[DO NOT MERGE] Allow online booking to be enabled from Locator

### PR DESCRIPTION
**Pull request by dwhenry**
Use `online_booking_enabled` flag to determine
if location is eligible for online booking instead
of hardcoded list of UID's.

**Ensure all locations are correct set with online booking enabled before merging**

### CODE REVIEW

File: lib/locations/location.rb [hunk 1]
```diff
@@ -1,3 +1,19 @@
 module Locations
-  Location = Struct.new(:id, :name, :address, :booking_location_id, :phone, :hours, :twilio_number, :lat_lng)
+  class Location
+    include ActiveModel::Model
```
Comment (benlovell) [inline lib/locations/location.rb]: What's the rationale for this? If you just want to support kwarg initialisation maybe just convert the `Struct` to `OpenStruct`? Also for the predicate method you could just pass a block to define methods within the scope of the struct

```ruby
Struct.new(:id, :blah) do
  def online_booking_enabled?
    # ...
  end
end
```

File: spec/support/helpers/test_location.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+class TestLocation
+  def self.london # rubocop:disable Metrics/MethodLength
```
Comment (benlovell) [inline spec/support/helpers/test_location.rb]: May as well just move the rule disable pragma to the top of the file?

**Comment by dwhenry** [comment]
@benlovell as discuss I am going to merge this as is. :)

**Comment by benlovell** [comment]
@dwhenry Did you see my latest comment?

**Pull request merged**